docs/advanced/metadata.md

```diff
@@ -0,0 +1,74 @@
+# Task Metadata
+
+::: warning
+Vitest exposes experimental private API. Breaking changes might not follow semver, please pin Vitest's version when using it.
+:::
+
+If you are developing a custom reporter or using Vitest Node.js API, you might want to pass down some data from tests that are running in different context to your reporter or custom Vitest handler.
+
+We cannot use [test context](/guide/test-context) for this because it is not serializable. Instead, Vitest provides `meta` property on any task (suite or test) that you can reuse between your tests and Node.js process. Beware that it is not a two-way communication, `meta` property can only be set form inside the test context. Changes inside Node.js context will not be reflected in your tests.
```

Comment:
```suggestion
To accomplish this, relying on the [test context](/guide/test-context) is not feasible since it cannot be serialized. However, with Vitest, you can utilize the `meta` property available on every task (suite or test) to share data between your tests and the Node.js process. It's important to note that this communication is one-way only, as the `meta` property can only be modified from within the test context. Any changes made within the Node.js context will not be visible in your tests.
```